modules/lotamePanoramaIdSystem.js

```diff
@@ -222,6 +268,9 @@ export const lotamePanoramaIdSubmodule = {
    * @returns {IdResponse|undefined}
    */
   getId(config, consentData, cacheIdObj) {
+    if (checkConfigHasErrorsAndReport(config)) {
+      return;
```

Comment:
just to confirm - the expectation when misconfigured is returning nothing?

Reply:
yes, it's much like Backbone validation... if there's an error, it's returned as a (truthy) string.